lib/katello/engine.rb

```diff
@@ -48,6 +48,13 @@ class Engine < ::Rails::Engine
           :oauth_secret => 'katello',
           :ca_cert_file => nil,
           :bulk_load_size => 1000
+        },
+        :candlepin_events => {
+          :broker_host => 'localhost',
+          :broker_port => 61_613,
```

Comment:
Is the underscore intentional?

Reply:
I just do what rubocop tells me to do like a good programmer ;)

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/NumericLiterals